checker/tests/nullness/TestInfer.java

```diff
@@ -11,7 +11,7 @@
     void bar(Object o) {}
 
     void foo() {
-        List<? extends Object> ls = new ArrayList<>();
+        List<@UnknownKeyFor ? extends Object> ls = new ArrayList<>();
```

Comment:
This is required because the defaults on a wildcard are set to the same as the type parameter.